modules/cluster-management/src/testFixtures/java/org/apache/ignite/internal/cluster/management/raft/TestClusterStateStorage.java

```diff
@@ -59,6 +66,24 @@ public class TestClusterStateStorage implements ClusterStateStorage {
     /** Prevents double stopping of the component. */
     private final AtomicBoolean stopGuard = new AtomicBoolean();
 
+    /**
+     * Returns a {@link TestClusterStateStorage} containing cluster state.
+     */
+    public static TestClusterStateStorage initializedClusterStateStorage() {
+        TestClusterStateStorage storage = new TestClusterStateStorage();
+
+        new ClusterStateStorageManager(storage).putClusterState(
```

Comment:
Are you creating a manager just to correctly put the value by key?
It's not obvious, maybe leave the documentation? Or just put by key without creating a manager?